src/modules/admin/components/Permissions/index.ts

```diff
@@ -1,3 +1 @@
-export { default as ColonyPermissionsEditDialog } from './ColonyPermissionsEditDialog';
-
-export { default as ColonyPermissionsAddDialog } from './ColonyPermissionsAddDialog';
+export { default as ColonyPermissionsAddDialog } from '~dashboard/ColonyPermissionsAddDialog';
```

Comment:
You don't need to export it from here anymore. 

You can remove the `index` file as well